### PR TITLE
Post schedule update

### PR DESCRIPTION
**Pull request by hypha**
- Since the 2023 photos are in the footer, I've replaced it with "call for volunteers" as the secondary CTA.. 
- For the primary CTA, as sponsorship is still officially ongoing, I've left it there, alongside ticket sales and schedule. 
- Not sure if the /programme is intended to be listed. Can remove if not. 
- Updated some extra milestone and wording in FAQ to reflect current status.

### CODE REVIEW

File: src/pages/index.astro [hunk 1]
```diff
@@ -54,13 +54,13 @@ deadlines = deadlines
...
-          EuroPython 2023 Photos
+          On-site Volunteer
         </Title>
 
         <ButtonLink
-          href="https://www.flickr.com/photos/europython/collections/72157722021442529/"
+          href="/volunteers"
         >
-          Highlights of EuroPython 2023
+          Join us in Making EuroPython 2024
```
Comment (egeakman) [inline src/pages/index.astro:63]: I'm not sure if "Making" would be the right word choice. Maybe "Running"?

Comment (hypha) [reply src/pages/index.astro:63]: Hmm I thought "making" involves broader and  more active collaboration. "Running" to me sounds a bit more bureaucratic. Does "Creating" or "Shaping" sound better to you?

Comment (egeakman) [reply src/pages/index.astro:63]: You are right. How about "Join us in Making EuroPython 2024 Happen"?

File: src/data/links.json [hunk 2]
```diff
@@ -3,6 +3,10 @@
     {
       "name": "Programme",
       "items": [
+        {
+          "name": "Overview",
+          "path": "/programme"
+        },
```
[the remark this answers is not in the document]
Comment (egeakman) [reply src/data/links.json:9]: I'm OK with keeping it this way.

By the way, should we reconsider the order of the items in "Programme" category/section? Or is the current order good enough? But this is probably for another PR.

File: src/pages/index.astro (hunk 1 above)
Comment (hypha) [reply src/pages/index.astro:63]: image
Haha I tested it before as well but I didn't like that on a full HD setting, the happen breaks off to the next line. I know it is not a biggie, but it bugged me a little :smile:

File: src/data/links.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (hypha) [reply src/data/links.json:9]: I thought the order was sort of OK. I did struggle a bit with where to put the /programme. Ofc, the schedule being on top might be nice. But I just don't know where else to stick it into. 

I am happy to include the order change here as well if you have suggestions.

File: src/pages/index.astro (hunk 1 above)
Comment (egeakman) [reply src/pages/index.astro:63]: Some alternatives could be:
- Help us Make EuroPython 2024 a Success
- Help us Make EuroPython 2024 a Reality
- Help Us Make EuroPython 2024 Memorable
- Help Us Make EuroPython 2024 Phenomenal

But "Shaping" is good as well

File: src/data/links.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (egeakman) [reply src/data/links.json:9]: I think it is good, but maybe could be better. I'd want to hear people's opinions on this. Let's discuss it in another issue or Discord.

**Review by egeakman — APPROVED** [review]
Thanks Raquel :heart:

**Comment by hypha** [comment]
@egeakman Feel free to merge if you are happy or tag extra reviewers (you know best whom to ask) <3

File: src/content/pages/faq.mdx [hunk 3]
```diff
@@ -95,6 +98,8 @@ We encourage and trust you to pick a fair ticket tier that fits your personal ci
 
 A. Yes, in-person participation applications for our [Financial Aid Programme](/finaid) was open until 19 May 2024. It is now closed after receiving a record-high number of applications. We are very proud to be able to support so many Pythonistas to attend the conference this year!
 
+[Financial Aid for Remote Tickets](/finaid#-free-remote-ticket) is open on 14 June, together with remote ticket sales. If you need support to attend the conference remotely, make sure to apply by 1 July 2024.
```
Comment (cybit) [inline src/content/pages/faq.mdx:101]:
```suggestion
[Financial Aid for Remote Tickets](/finaid#-free-remote-ticket) opened on 14 June, together with remote ticket sales. If you need support to attend the conference remotely, make sure to apply by 1 July 2024.
```
```suggestion
[Financial Aid for Remote Tickets](/finaid#-free-remote-ticket) is open on 14 June, together with remote ticket sales. If you need support to attend the conference remotely, make sure to apply by 1 July 2024.
```

File: src/pages/index.astro [hunk 4]
```diff
@@ -54,13 +54,13 @@ deadlines = deadlines
...
-          EuroPython 2023 Photos
+          On-site Volunteer
         </Title>
 
         <ButtonLink
-          href="https://www.flickr.com/photos/europython/collections/72157722021442529/"
+          href="/volunteers"
         >
-          Highlights of EuroPython 2023
+          Join us in Shaping EuroPython 2024
```
Comment (cybit) [inline src/pages/index.astro:63]: I expect the uppercase "Shaping" to be on purpose?

**Review by cybit — APPROVED** [review]
Small fry only.

File: src/content/pages/faq.mdx (hunk 3 above)
Comment (hypha) [reply src/content/pages/faq.mdx]: fixed with thanks!

File: src/pages/index.astro (hunk 4 above)
Comment (hypha) [reply src/pages/index.astro]: Yes, but I also wanted to make us into Us. Now fixed. Thanks for spotting it.

**Pull request merged**